kasa/tapo/tapobulb.py

```diff
@@ -17,6 +17,11 @@ class TapoBulb(TapoDevice, SmartBulb):
     Documentation TBD. See :class:`~kasa.smartbulb.SmartBulb` for now.
     """
 
+    @property
+    def is_bulb(self) -> bool:
+        """Return True if the device is a bulb."""
+        return True
+
```

Comment:
Should not be needed, but handled by the base class. See my comment below.

Reply:
Removed